# Optitrack rigid body
int32                 id
bool                  tracking_valid
float64               mrk_mean_error
geometry_msgs/Pose    pose
geometry_msgs/Point[] markers
